### PR TITLE
Files: encode local path on Windows

### PR DESCRIPTION
**Pull request by songgao**
This encodes the local path in the pop up from the Files tab.

### CODE REVIEW

File: go/libkb/util.go [hunk 1]
```diff
@@ -1052,15 +1052,24 @@ func IsKBFSAfterKeybasePath(afterKeybase string) bool {
...
+		return afterMount
+	}
+
+	// Encode path names for Windows
+	elems := strings.Split(afterMount, "/")
+	for i, elem := range elems {
+		elems[i] = EncodeKbfsNameForWindows(elem)
 	}
-	return afterMount
+	afterMount = strings.Join(elems, "/")
```
Comment (strib) [inline go/libkb/util.go:1070]: Why re-join with `/` instead of just `\`, and skip the `Replace` call?

**Review by strib — APPROVED** [review]

File: go/libkb/util.go (hunk 1 above)
Comment (songgao-zoom) [reply go/libkb/util.go]: good point! updated

**Comment by strib** [comment]
> Hmm CI keeps fail on seemingly irrelevant tests. Good to go?

Is this a question for me?  Still looks good to me.  And looks like master is failing on those tests too.

**Comment by songgao-zoom** [comment]
> > Hmm CI keeps fail on seemingly irrelevant tests. Good to go?
> 
> Is this a question for me? Still looks good to me. And looks like master is failing on those tests too.

Thanks! Yeah just wanted an ack from someone before I hit the green button.

**Pull request merged**